VPredoi_Abstract_EGU25_ESSI3-3.txt

```diff
@@ -10,15 +10,30 @@ state-of-the-art analyses. ESMValTool can be used for tasks ranging from monitor
 simulations to analysis for scientific publications such as the IPCC reports, including reproducing results from
 previously published scientific articles as well as allowing scientists to produce new analysis results.
 
-Software reliability and reproducibility is of the essence for ESMValTool. Our main strategy for ensuring reliability
-is modular, integrated, and tested design. This comes back at various levels of the tool. We try to separate commonly
-used functionality from “one off” code, and make sure that commonly used functionality is covered by unit and integration
-tests, while we rely on regression testing for everything else. We also use comprehensive end-to-end testing for all
-our “recipes” before we release new versions. Here we describe the testing infrastructure for both ESMValCore, the autonomous
-computational engine framework that runs the analyses provided by ESMValTool, and for the package comprising
-scientific diagnostics and metrics. Our testing infrastructure ranges from basic unit tests to tools that smartly handle
+ESMValTool is FAIR research software because it is: Findable - it is published in community registries, such
```

Comment:
Maybe format the four properties of FAIR as a list for improved readability?

Reply:
I wanted to do exactly that - good call, bud! But then I thought the  abstract may "look" a lot bigger than it is, and was afraid it'd get rejected at submission. It's already edging towards the 500 words limit

Reply:
FAIR a list in https://github.com/valeriupredoi/EGU2025/pull/2/commits/0b083f3dbe43336c9dcb54f2bb41050559ffcb15